Parse/TPTP.hpp

```diff
@@ -352,9 +351,8 @@ class TPTP
   unsigned lineNumber(){ return currentFile.lineNumber; }
   std::string currentPath(){ return currentFile.path; }
 
-  static Map<int,std::string>* findQuestionVars(unsigned questionNumber) {
-    auto res = _questionVariableNames.findPtr(questionNumber);
-    return res ? *res : nullptr;
+  static Map<unsigned,std::string>* findQuestionVars(unsigned questionNumber) {
+    return _questionVariableNames.findPtr(questionNumber);
```

Comment:
So a danger would be if someone requests `findQuestionVars`, keeps the pointer stored with them, TPTP later inserts to `_questionVariableNames` some more, the `DHMap` grows, `Map`s inside it get `move`ed and the old pointer gets invalidated.

If you considered this option and have checked this cannot happen, all is fine!